hudi-cli/src/main/java/org/apache/hudi/cli/HoodieCLI.java

```diff
@@ -80,8 +82,10 @@ public static boolean initConf() {
   }
 
   public static void initFS(boolean force) throws IOException {
-    if (fs == null || force) {
-      fs = (tableMetadata != null) ? tableMetadata.getFs() : FileSystem.get(conf);
+    if (storage == null || force) {
+      storage = (tableMetadata != null)
+          ? tableMetadata.getHoodieStorage()
```

Comment:
rename: just `getStorage()`. not `getHoodieStorage()`. We don't need that extract qualifier